samples/net/coap_client/src/coap-client.c

```diff
@@ -36,20 +36,27 @@ struct coap_pending pendings[NUM_PENDINGS];
 struct coap_reply replies[NUM_REPLIES];
 struct k_delayed_work retransmit_work;
 
-#if defined(CONFIG_NET_MGMT_EVENT)
-static struct net_mgmt_event_callback cb;
-#endif
-
 static const char * const test_path[] = { "test", NULL };
 
+/* define semaphores */
+K_SEM_DEFINE(COAP_sem, 0, 1);	/* starts off "available" */
```

Comment:
I suppose this should be named in lower and do not append comments after the code.